include/fg/plot.h

```diff
@@ -253,58 +253,58 @@ class Plot {
         FGAPI void setMarkerSize(const float pMarkerSize);
 
         /**
-           Get the OpenGL buffer object identifier for vertices
+           Get the buffer identifier for vertices
 
-           \return OpenGL VBO resource id.
+           \return vertex buffer resource id.
          */
         FGAPI unsigned vertices() const;
 
         /**
-           Get the OpenGL buffer object identifier for color values per vertex
+           Get the buffer identifier for color values per vertex
 
-           \return OpenGL VBO resource id.
+           \return colors buffer resource id.
          */
         FGAPI unsigned colors() const;
 
         /**
-           Get the OpenGL buffer object identifier for alpha values per vertex
+           Get the buffer identifier for alpha values per vertex
 
-           \return OpenGL VBO resource id.
+           \return alpha values buffer resource id.
          */
         FGAPI unsigned alphas() const;
 
         /**
-           Get the OpenGL buffer object identifier for markers sizes, per vertex
+           Get the buffer identifier for per vertex marker sizes
 
-           \return OpenGL VBO resource id.
+           \return marker sizes buffer resource id.
          */
         FGAPI unsigned markers() const;
```

Comment:
I am not entirely sure what this is being used for, so I don't know what is appropriate. I just noticed that they were not consistent.

@shehzan10 thoughts ?